schema/v2/faction_schema.json

```diff
@@ -5,15 +5,54 @@
   "items": {
     "type": "object",
     "additionalProperties": false,
+    "allOf": [
+      {
+        "title": "Faction codes must be valid for the side.",
+        "type": "object",
+        "oneOf": [
+          {
+            "properties": {
+              "side_code": {
+                "const": "corp"
+              },
+              "code": {
+                "enum": [
+                  "haas-bioroid",
+                  "jinteki",
+                  "nbn",
+                  "neutral-corp",
+                  "weyland-consortium"
+                ]
+              }
+            }
+          },
+          {
+            "properties": {
+              "side_code": {
+                "const": "runner"
+              },
+              "code": {
+                "enum": [
+                  "adam",
+                  "anarch",
+                  "apex",
+                  "criminal",
+                  "neutral-runner",
+                  "shaper",
+                  "sunny-lebeau"
+                ]
+              }
+            }
+          }
+        ]
+      }
+    ],
     "properties": {
       "code": {
         "minLength": 1,
         "type": "string"
       },
-      "color": {
-        "description": "Faction color in hex triplet format. This can be used to display faction-specific information in the appropriate color on an HTML webpage.",
-        "maxLength": 6,
-        "minLength": 6,
```

Comment:
I actually used these values as the embed color for Sahasrara's card outputs. Would it be possible to keep this information on NRDB?

Reply:
i have a preference for keeping color information and other formatting info outside.  I don't even like that we have HTML tags in the text and would prefer something like markdown to keep things a bit more format-flexible.

I am happy to throw these back in for now, though, if you like.